src/pages/taejoon/List/List.scss

```
@@ -0,0 +1,89 @@
+
+
+.container-List {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    
+}
+.header {
+    background-color: 	#f4f3ec;
+    display: flex;
+    justify-content: space-between;
+    width: 90%;
+    margin-left: 5%;
+    
+}
+.main_logo {
+    font-family: 'Dancing Script', cursive;
+    font-size: 1.5rem;
+    text-decoration: none;
+    color: black;
+    padding: 1% 2% 1% 2%;
```

```suggestion
    padding: 1% 2%;
```
이렇게만 써도 되는걸로 알고 있습니다.